ilastik/applets/objectClassification/objectClassificationGui.py

```diff
@@ -163,20 +164,24 @@ def __init__(self, parentApplet, op):
         self._colorTable16_forpmaps[15] = QColor(Qt.black).rgba() #for objects with NaNs in features
         
         # button handlers
-        self._interactiveMode = False
+        self.interactiveMode = False
```

Comment:
uhm, was this a bug?? the setters are also still there, and the logic in them isn't needed anymore?